src/main/java/lottogame/model/lotto/Lotto.java

```diff
@@ -0,0 +1,46 @@
+package lottogame.model.lotto;
+
+import static java.lang.String.join;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class Lotto {
+    public static final int LOTTO_NUM_COUNT = 6;
+
+    private final Set<LottoNum> lottoNums;
+
+    public Lotto(Set<LottoNum> lottoNums) {
```

Comment:
```suggestion
    public Lotto(int... lottoNums) {
    
    public Lotto(String commaText) {    
    
    public Lotto(LottoNum... lottoNums) { 
```
이와 같이 여러 부 생성자를 추가하는 것은 어떨까?